crypto/rand/rand_lib.c

```diff
@@ -749,31 +749,52 @@ void RAND_add(const void *buf, int num, double randomness)
  * the default method, then just call RAND_bytes().  Otherwise make
  * sure we're instantiated and use the private DRBG.
  */
-int RAND_priv_bytes(unsigned char *buf, int num)
+int rand_priv_bytes_ex(OPENSSL_CTX *ctx, unsigned char *buf, int num)
```

Comment:
IMHO the addition of the _ex functions in rand_lib.c should be a separate commit preceding this commit.